homeassistant/components/speedtestdotnet/manifest.json

```diff
@@ -1,7 +1,8 @@
 {
   "domain": "speedtestdotnet",
-  "name": "Speedtest.net",
+  "name": "Speedtestdotnet",
```

Comment:
This change is incorrect. Their brand does not use the `dot` in the name. Please revert this to the previous value.